app/src/main/java/com/hedvig/app/feature/home/ui/claimstatus/data/RelatedContractTypeData.kt

```diff
@@ -0,0 +1,16 @@
+package com.hedvig.app.feature.home.ui.claimstatus.data
+
+import androidx.compose.runtime.Composable
+import com.hedvig.android.owldroid.graphql.HomeQuery
+
+@JvmInline
+value class RelatedContractTypeData(val text: String) {
+    companion object {
+        @Composable
+        fun fromClaimStatus(claimStatus: HomeQuery.ClaimStatus): RelatedContractTypeData {
+            // TODO Is displayName appropriate?
```

Comment:
For example for Swedish house we get "Home Insurance Renter" but is this what we're supposed to show to the user? Do we have a nicer human-readable version of these, or are these the appropriate ones?

Reply:
I think this is indeed the appropriate one 🙂